nilearn/decoding/tests/test_searchlight.py

```diff
@@ -3,11 +3,15 @@
 # Author: Alexandre Abraham
 
 import numpy as np
+import pytest
 from nibabel import Nifti1Image
+from searchlight import SearchLight
```

Comment:
I do not think this will work.

Also there is already a `from nilearn.decoding import searchlight` below that should do the trick.